docs/docsite/rst/user_guide/playbook_pathing.rst

```diff
@@ -32,9 +33,7 @@ i.e ::
     play search path is playdir/{files|vars|templates}/, playdir/.
 
 
-The current working directory (cwd) is not searched. If you see it, it just happens to coincide with one of the paths above.
-If you `include` a task file from a role, it  will NOT trigger role behavior, this only happens when running as a role, `include_role` will work.
-A new variable `ansible_search_path` var will have the search path used, in order (but without the appended subdirs). Using 5 "v"s (`-vvvvv`) should show the detail of the search as it happens.
+By default, Ansible does not search the current working directory (cwd) unless it happens to coincide with one of the paths above. If you `include` a task file from a role, it  will NOT trigger role behavior, this only happens when running as a role, `include_role` will work. A new variable `ansible_search_path` var will have the search path used, in order (but without the appended subdirs). Using 5 "v"s (`-vvvvv`) should show the detail of the search as it happens.
```

Comment:
hard to tell in diffs but do we use the abbreviation (cwd) elsewhere and/or is everyone but me familiar with it?  If not, then just nix it.

Reply:
Good catch. The command for "where on the filesystem am I?" is `pwd` rather than `cwd`, so I think removing this would be good.